localstack-core/localstack/services/kms/provider.py

```diff
@@ -737,7 +756,7 @@ def _generate_data_key_pair(
             "PublicKey": crypto_key.public_key,
         }
 
-    @handler("GenerateDataKeyPair")
```

Comment:
i had to remove the empty handlers, as these somehow interfered with the tests in `asf_utils.py` - didn't investigate further, I am sure we could solve this and keeping the handler. On the other hand the handler doesn't do anything here :)